app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        name_str = f"Name: {self.name}"
+        health_str = f"Health: {self.health}"
+        hidden_str = f"Hidden: {self.hidden}"
+        return f"{{{name_str}, {health_str}, {hidden_str}}}"
```

Comment:
too complicated, you can just pass them all in a single return of some structure

Reply:
no easy options!!!